src/CommandingOfficers/AWBW/YC/SonjaDSBW.java

```diff
@@ -29,15 +29,15 @@ private static class instantiator extends CommanderInfo
     private static final long serialVersionUID = 1L;
     public instantiator()
     {
-      super("Cyrus", UIUtils.SourceGames.AWBW, UIUtils.YC);
+      super("Sonja", UIUtils.SourceGames.AWBW, UIUtils.YC, "DSW");
       infoPages.add(new InfoPage(
-            "Cyrus (AWDS Sonja for AWBW)\n"
+            "AWDS Sonja for AWBW\n"
```

Comment:
This change is based on a recent conversation with Jay.
I wasn't particularly attached to the "Cyrus" thing anyhow, and this is clearer.